### PR TITLE
[Contribution] abuseipdb-get-blacklist confidenceMinimum addition

### PR DESCRIPTION
**Pull request by Phoenix-912**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: add confidenceMinimum level 

## Description
This pull request allows to add the `confidenceLevel` into the request made by XSOAR to get a blacklist from AbuseIPDB. The confidence level is the level from which to retrieve IP addresses known by AbuseIPDB to be associated to Threat Actors.

More information about the API here: https://docs.abuseipdb.com/#blacklist-endpoint.

## Screenshots
N/A

## Minimum version of Demisto
- 5.0.0

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No
          - Further details: the `confidenceMinimum` was default to 100 whenever not specified as visible here: https://docs.abuseipdb.com/#blacklist-caching. So this new argument should not impact playbook previously developped as it is only optional.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by reutshal** [comment]
@roysagi We will probably take this, mind taking a look at the description in the yml, README and RN? Thanks

**Comment by reutshal** [comment]
@Phillips-Chris Thanks for the contribution!
Why did you remove this file? 
image
Please revert this change

**Review by reutshal — APPROVED** [review]

File: Packs/AbuseDB/Integrations/AbuseDB/AbuseDB.yml [hunk 1]
```diff
@@ -251,6 +251,13 @@ script:
       - 'false'
       required: false
       secret: false
+    - default: false
+      defaultValue: '100'
+      description: 'The Minimum confidence required of IPs to retrieve. Default is 100.'
```
Comment (roysagi) [inline Packs/AbuseDB/Integrations/AbuseDB/AbuseDB.yml:256]:
```suggestion
      description: The Minimum confidence required for the retrieved IPs.
```
No need to note the default version, the UI will do it automatically.

File: Packs/AbuseDB/Integrations/AbuseDB/README.md [hunk 2]
```diff
@@ -303,6 +303,11 @@
 <td style="width: 481px;">Maximum number of IPs to retrieve, default is 50</td>
 <td style="width: 90px;">Optional</td>
 </tr>
+<tr>
+<td style="width: 169px;">confidence</td>
+<td style="width: 481px;">Minimum confidence of IPs to retrieve, default is 100</td>
```
Comment (roysagi) [inline Packs/AbuseDB/Integrations/AbuseDB/README.md:308]:
```suggestion
<td style="width: 481px;">The Minimum confidence required for the retrieved IPs, default is 100</td>
```

File: Packs/AbuseDB/ReleaseNotes/1_0_2.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### AbuseIPDB
+Added the **confidence** argument to the ***abuseipdb-get-blacklist*** command.
```
Comment (roysagi) [inline Packs/AbuseDB/ReleaseNotes/1_0_2.md:4]:
```suggestion
- Added support for the *confidence* argument to the ***abuseipdb-get-blacklist*** command.
```

**Pull request merged**